packages/docs-components/src/knobs/defaultComponents.tsx

```diff
@@ -34,6 +34,16 @@ const KnobBoolean: React.FunctionComponent<KnobComponentProps> = props => (
   />
 )
 
+const KnobNumber: React.FunctionComponent<KnobComponentProps> = props => (
```

Comment:
do we really need this? can't we have string, number and boolean together in one `KnobPrimitive`?

Reply:
Yep, because it renders different components:
- `KnobBoolean` - `input` with `type="checkbox"`, will be replaced with `Checkbox` later
- `KnobNumber` - `input` with `type="number"`, value is parsed with `parseInt()`
- `KnobString` - usual `input`